client/components/domains/register-domain-step/index.jsx

```diff
@@ -579,6 +575,11 @@ class RegisterDomainStep extends React.Component {
 						-1,
 						this.props.analyticsSection
 					);
+
+					this.setState( {
```

Comment:
Without this, the wpcom subdomain suggestion would remain in the loading state when an error occurs. Of course, it's super rare for that to happen, but 🤷‍♂️

Reply:
Nice catch. Thank you.